src/adapter/services/LegacyPersistenceAdapter.js

```diff
@@ -0,0 +1,29 @@
+import objectUtils from 'objectUtils';
```

Comment:
Could we use ES6 class for this implementation? Since its a new implementation

Reply:
And we need the copyright message at the top

Reply:
> Could we use ES6 class for this implementation? Since its a new implementation

If memory serves, the legacy framework expects a regular function and won't work with an ES6 constructor function. So, we can return a ES6 class here, but the implementation passed to the framework will need to be a function that wraps the ES6 class.

Reply:
So is there any point in doing this then?

Reply:
I think so, because it's a small change and it keeps out code consistent. 

The implementation function would just do something like this (I think?):
```JavaScript
    implementation: function legacyPersistenceAdapter () {
        return new LegacyPersistenceAdapter();
    }
```

Reply:
ok!